cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/ServiceImpl.kt

```diff
@@ -164,6 +164,7 @@ private class ServiceImpl(
 
                 networkClient.getConsentStatus(consentStatusParamReq)
                     .executeOnRight { consentStatusResponse ->
+                        dataStorage.updateLocalDataVersion()
```

Comment:
Here, for example, I'd just set instead of using the dedicated `updatedLocalDataVersion()`

```kotlin
dataStorage.localDataVersion = HARDCODED_LOCAL_DATA_VERSION
```

Reply:
we have to encapsulate the logic of updating `localDataVersion` to the `HARDCODED_LOCAL_DATA_VERSION`